README.md

```diff
@@ -42,11 +42,14 @@ At last, visit `localhost:3001`.
 To run locally in production mode run
 
     $ npm run dist
-    $ NODE_ENV=production APP_ENV=[environment variable] npm start
+    $ clientID=[client id] clientSecret=[client secret] NODE_ENV=production APP_ENV=[environment variable] npm start
 
 and visit `localhost:3001`.
 NOTE: Currently the only working `environment variable` is `development`.
 
+## Client Id and Secret
```

Comment:
But the outsider can ask anyone here and the person can help to deliver the messages. I think it is clear than just coworkers.